### PR TITLE
Feature tvos documentation

### PR DESCRIPTION
**Pull request by shokinn**
- [x] have read the [CONTRIBUTING.md](./CONTRIBUTING.md) file
- [ ] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [x] updated documentation if needed
- [ ] updated CHANGELOG.md

## Summary

- Documentation
  - Updated `docs/about/clients.md`, added `tvOS` as client.
  - Updated `docs/usage/connect/apple.md`, added Installation and configuration documentation for tvOS.
  - Updated `hscontrol/templates/apple.go` to also display the tvOS configuration at `/apple` on a deployed headscale instance.

### CODE REVIEW

File: hscontrol/templates/apple.go [hunk 1]
```diff
@@ -106,43 +154,108 @@ func Apple(url string) *elem.Element {
...
+						attrs.Props{
+							attrs.Href: "https://apps.apple.com/app/tailscale/id1470499037",
+						},
+						elem.Text("App store"),
+					),
+				),
+				elem.Li(
+					nil,
+					elem.Text(
+						"Go Settings (the apple tvOS settings) > Apps > Tailscale",
```
Comment (nblock) [inline hscontrol/templates/apple.go:229]: Open Settings …

**Review by nblock — CHANGES_REQUESTED** [review]
Thanks for your contribution!

I left a few minor suggestions for the docs part.

**Comment by shokinn** [comment]
Sounds fair.
I've added the requested changes.

**Review by nblock — APPROVED** [review]

File: hscontrol/templates/apple.go [hunk 2]
```diff
@@ -106,43 +154,108 @@ func Apple(url string) *elem.Element {
...
 					),
 				),
+				elem.Li(nil,
+					elem.Text("Return to the tvOS Home screen"),
+				),
+				elem.Li(nil,
+					elem.Text("Open Tailscale"),
+				),
+				elem.Li(nil,
+					elem.Text("Select \"Install VPN configuration\""),
```
Comment (kradalby) [inline hscontrol/templates/apple.go:248]:
```suggestion
					elem.Text(`Select "Install VPN configuration"`),
```

for the places its escaped, makes it a bit easier to read.

**Pull request merged**